server/data/restClient.ts

```diff
@@ -1,24 +1,47 @@
-import superagent, { SuperAgentRequest } from 'superagent'
+import superagent, { Response, ResponseError } from 'superagent'
```

Comment:
This looks like I've made lots of changes to `restClient`, but this is actually a verbatim copy and paste of `restClient` from PLP UI